.github/workflows/validations.yml

```
@@ -3,101 +3,330 @@
+          PUPPETEER_SKIP_DOWNLOAD=true pnpm install --frozen-lockfile
+      
+      - name: Install Playwright Chromium browser (with deps)
+        if: steps.playwright-cache.outputs.cache-hit != 'true'
+        run: pnpm exec playwright install --with-deps chromium
+      
+      - name: Install Playwright Chromium browser deps only
+        if: steps.playwright-cache.outputs.cache-hit == 'true'
+        run: pnpm exec playwright install-deps chromium
+      
+      - name: Run Rubocop
+        if: matrix.ci_node_index == 0
```

actionlint (1.7.7)

property "ci_node_index" is not defined in object type {}

(expression)